examples/ecs/variables.tf

```
@@ -148,3 +148,9 @@ variable "ipfs_default_log_level" {
   description = "IPFS default log level"
   default     = "info"
 }
+
+variable "existing_ipfs_peers" {
```

```suggestion
variable "existing_ipfs_peer" {
```

---

I intentionally added the plural "peers" in the top-level module's variable name because it applies to all the deployments under the module (i.e. depending on the value of `var.deployments` passed in during the apply).

It's called `existing_ipfs_peer` in `modules/ecs/variables.tf` and `existing_peer` in `modules/ecs/ipfs/variables.tf` in keeping with that terminology.

---

Ok I see. better suggestion
```suggestion
variable "use_existing_ipfs_peer_identities" {
```

---

Cool, sounds good! 👍🏼